src/javascript/app/pages/resources/trading_times/trading_times.ui.js

```diff
@@ -30,13 +30,26 @@ const TradingTimesUI = (() => {
         }
 
         const date = moment.utc();
-        $date.attr('data-value', toISOFormat(date));
+        const isoFormattedDate = toISOFormat(date);
+        $date.attr('data-value', isoFormattedDate);
         DatePicker.init({
             selector: '#trading-date',
             minDate : 0,
             maxDate : 364,
         });
         $date.val(localize('Today'));
+        // Create a label to be friendlier
+        const $label = $('label[for=trading-date]');
+        $label.append('<span class=\'ux-date foot-note\'></span>');
+        if ($date.val() === '') {
+            $('span.ux-date').text(localize('Today'));
+            $date.val(isoFormattedDate);
+            $date.attr('value', isoFormattedDate);
+        }
+        $date.change(() => {
+            $('span.ux-date').text(moment($date.val()).format('LL'));
```

Comment:
do we need `.utc` here too?

Reply:
I couldn't quite catch that, you mean line 50?

Reply:
yes, something like `moment.utc($date.val()`

Reply:
I changed my time zone drastically to see if it changes. It didn't. I think because we are using a date field, changing time doesn't affect its behavior. So the only time that this can come to play is when we are initiating a date field, and that is passed through a UTC filter.

Reply:
as discussed and tested with @binary-hamedanchi it seems you're right and it only affects things like `.valueOf()` which need the timezone, no change needed then